packages/scripts/index.js

```diff
@@ -30,21 +33,50 @@ const getOutputDir = (output) => {
   return outputDirMatch.length > 1 ? outputDirMatch[1] : output;
 };
 
+// Runs typescript compiler for type checking and emitting declarations in a separate process to run in parallel with
+// the bundler.
+const runTsc = (options) => {
+  const tscBin = path.resolve(`./node_modules/.bin/tsc`);
+  return exec(`${tscBin} --emitDeclarationOnly`)
+    .then(() => ora().succeed(chalk.green.bold('type checking done')))
+    .catch(({ stderr, stdout }) => {
+      console.log();
+      if (stdout) {
+        console.log(stdout);
+      }
+      if (stderr) {
+        console.error(stderr);
+      }
+    })
+    .finally(() => {
+      options.forEach((option) => {
+        const fixTSPathRewritePlugin = option.plugins.find((plugin) => plugin.name == 'fixTSPathRewrite');
+        if (fixTSPathRewritePlugin) {
+          fixTSPathRewritePlugin.writeBundle();
+        }
```

Comment:
What does this do?

Reply:
It [re-writes](https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/rollup.config.js#L14-L29) typescript's auto path aliasing with references. 

This needs to be run after typescript finishes generating `.d.ts` files and that is why it is run here. This shall be removed after removing aliases from spinnaker modules.